source/common/http/http1/BUILD

```diff
@@ -108,11 +108,21 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "legacy_parser_lib",
-    srcs = ["legacy_parser_impl.cc"],
-    hdrs = ["legacy_parser_impl.h"],
-    external_deps = ["http_parser"],
+    srcs = [
+        "legacy_parser_impl.cc",
+        "legacy_permissive_parser_impl.cc",
```

Comment:
In general with bazel the preference is to create fine grained targets. In this case the legacy_permissive_parser_impl.cc will be in its own library

Reply:
Done.